easybuild/easyblocks/p/psi.py

```diff
@@ -138,7 +156,7 @@ def sanity_check_step(self):
         """Custom sanity check for PSI."""
         custom_paths = {
             'files': ['bin/psi%s' % self.version.split('.')[0]],
-            'dirs': ['include', 'share/psi'],
+            'dirs': ['share/psi'],
```

Comment:
They no longer install the headers like this. No idea if that's intentional or a bug in their new build system